### ISSUE TITLE
Animate page transition stops working.

### ISSUE THREAD

**Issue by THElegend5** [tachiyomiorg/tachiyomi#8296]
### Steps to reproduce

1. Go to any manga with multiple downloaded chapters and about 50 - 100 pages each.
2. Start reading any chapter except for the last with page transition animations turned on.

### Expected behavior

Page transition happen all the way through the chapter.

### Actual behavior

Page transition stops working about 30 - 40 page into a chapter but I think this happens after a certain amount of time passes and isn't page dependent.

### Tachiyomi version

Preview r5104

### Android version

Android 12

### Device

Redmi Note 9 Pro Max.

### Other details

This make it so that webtoon are borderline unreadable and manga is barely manageable.
Note that if you use touch instead of volune buttons to navigate, the page transitions till the point where you pull the next page after that it jumps to next page instead of smoothly scrolling to it like normal.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.13.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by GGG-KILLER** [comment]
Reproduction steps are different in 0.14.1 as the chapter doesn't even need to be long. I've reproduced it in a 9 page chapter.

As mentioned in #8352, disabling page transition animations and using the tap regions does not solve this issue.

I've recorded [a video](https://user-images.githubusercontent.com/5892127/198864207-b93def35-0968-4501-8496-c8a42e61fd48.webm) of it happening in 7th page of a 35 page chapter.

**Comment by THElegend5** [comment]
> Reproduction steps are different (bar is ***much*** lower) in 0.14.1 as the chapter doesn't even need to be long. I've reproduced it in a 9 page chapter.
> 
> As mentioned in #8352, disabling page transition animations and using the tap regions does not solve this issue.
> 
> I've recorded [a video](https://user-images.githubusercontent.com/5892127/198864207-b93def35-0968-4501-8496-c8a42e61fd48.webm) of it happening in 7th page of a 35 page chapter.

Yeah as I said I think it happens after a certain amount of time as long as the chapter which is being read is not the last unread chapter in which case this doesn't happen for me.

**Comment by GGG-KILLER** [comment]
> Yeah as I said I think it happens after a certain amount of time as long as the chapter which is being read is not the last unread chapter in which case this doesn't happen for me.

I don't think it has anything to do with the amount of time, I have even reinstalled the app (after trying to downgrade but failing due to the DB migrations I assume) and the issue happened as soon as I opened the app.

**Comment by GGG-KILLER** [comment]
An update: the issue is only the transition animation itself, if I do the animation with my finger (by swiping across the entire screen without lifting it), the page transition progresses much further than if I swipe just a little and let the animation do the rest.

**Comment by sillysack69** [comment]
I'm having this same problem though I would describe it as such: the beginning pages of a chapter scroll smoothly (as they should) but once a certain amount of pages is reached, the scrolling experience degrades into a crude distracting jerky motion. This was not present prior to 0.14.0.

**Comment by GGG-KILLER** [comment]
> I'm having this same problem though I would describe it as such: the beginning pages of a chapter scroll smoothly (as they should) but once a certain amount of pages is reached, the scrolling experience degrades into a crude distracting jerky motion. This was not present prior to 0.14.0. Also I also almost completely neglected to mention that this only applies to downloaded chapters. Online reading works just fine.

In my case online reading has the same issue, I initially discovered it reading an online chapter actually.
And I've also had it happen on the 2nd page of a chapter.
So I think the problem is more widespread than that

**Comment by arkon** [comment]
Is this any better in [r5172](https://github.com/tachiyomiorg/tachiyomi-preview/releases/tag/r5172)?

**Comment by GGG-KILLER** [comment]
> Is this any better in [r5172](https://github.com/tachiyomiorg/tachiyomi-preview/releases/tag/r5172)?

Have tested downloaded chapters where I had the issue previously and it has been solved.
Also tested on online chapters and haven't been able to reproduce the issue.

@THElegend5 and @sillysack69 can you test on your end?

**Comment by sillysack69** [comment]
> > Is this any better in [r5172](https://github.com/tachiyomiorg/tachiyomi-preview/releases/tag/r5172)?
> 
> Have tested downloaded chapters where I had the issue previously and it has been solved. Also tested on online chapters and haven't been able to reproduce the issue.
> 
> @THElegend5 and @sillysack69 can you test on your end?

Yeah. It seems to be working fine in [r5172](https://github.com/tachiyomiorg/tachiyomi-preview/releases/tag/r5172). I see smooth scrolling on downloaded chapters now too. What exactly was causing the problem @arkon?

**Comment by GGG-KILLER** [comment]
> Yeah. It seems to be working fine in [r5172](https://github.com/tachiyomiorg/tachiyomi-preview/releases/tag/r5172). I see smooth scrolling on downloaded chapters now too. What exactly was causing the problem @arkon?

I'd need some confirmation, but I believe this is what fixed it https://github.com/tachiyomiorg/tachiyomi/commit/f9c25b350ed1eb5cb418e804a829f40ecfb0af2e

**Comment by arkon** [comment]
It was https://github.com/tachiyomiorg/tachiyomi/commit/fc184f1cfa929328c41b0ec374503afe204d6c15

**Comment by THElegend5** [comment]
Yeah same seems to have been fixed.

**Comment by GGG-KILLER** [comment]
> It was [fc184f1](https://github.com/tachiyomiorg/tachiyomi/commit/fc184f1cfa929328c41b0ec374503afe204d6c15)

Oh, I see, that is indeed the issue
Disabling download ahead also fixes the freezing issue (as a temporary fix while a new version isn't released with that commit)

**Comment by sillysack69** [comment]
> > It was [fc184f1](https://github.com/tachiyomiorg/tachiyomi/commit/fc184f1cfa929328c41b0ec374503afe204d6c15)
> 
> Oh, I see, that is indeed the issue Disabling download ahead also fixes the freezing issue (as a temporary fix while a new version isn't released with that commit)

I tried doing that. Works as advertised! Guess that feature download ahead had some bug. BTW still waiting on the 2 page merge feature. No rush.
